vnext/PropertySheets/External/Microsoft.ReactNative.WAS.props

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
nit: The `WAS` acronym used by the property sheets isn't used elsewhere, so it be a bit confusing to interpret on close glace. Could we use WindowsAppSdk or Reunion?

Reply:
yep, done.